core/sail/inferencer/src/main/java/org/eclipse/rdf4j/sail/inferencer/fc/AbstractForwardChainingInferencerConnection.java

```diff
@@ -174,7 +174,9 @@ protected void doInferencing()
 			logger.debug("iteration " + iteration + " done; inferred " + nofInferred + " new statements");
 			totalInferred += nofInferred;
 		}
-		logger.info("{} inferred {} new statements", this.getClass().getName(), totalInferred);
+		if (totalInferred > 0) {
```

Comment:
This is an improvement. What is the usecase for it being info level and not debug (with the more verbose logging in the loop above it going down to trace)?